spec/lib/post_action_creator_spec.rb

```diff
@@ -271,18 +271,40 @@
   end
 
   describe "take_action" do
-    before { PostActionCreator.create(Fabricate(:user), post, :inappropriate) }
+    it "will hide the post" do
+      PostActionCreator
+        .new(Fabricate(:moderator), post, PostActionType.types[:spam], take_action: true)
+        .perform
+        .reviewable
+      expect(post.reload.hidden).to eq(true)
+    end
+
+    context "when there is another reviewable on the post" do
+      before { PostActionCreator.create(Fabricate(:user), post, :inappropriate) }
+
+      it "will agree with the old reviewable" do
+        reviewable =
+          PostActionCreator
+            .new(Fabricate(:moderator), post, PostActionType.types[:spam], take_action: true)
+            .perform
+            .reviewable
+        scores = reviewable.reviewable_scores
+        expect(scores[0]).to be_agreed
+        expect(scores[1]).to be_agreed
```

Comment:
I know this is not your code. Just pointing out the `#all` matcher which would probably read nicely here:

```suggestion
        expect(reviewable.reviewable_scores).to all(be_agreed)
```